diff --git a/src/components/TheAddEditSailorDialogComponent.vue b/src/components/TheAddEditSailorDialogComponent.vue
index 0cafbea..969a8cd 100644
--- a/src/components/TheAddEditSailorDialogComponent.vue
+++ b/src/components/TheAddEditSailorDialogComponent.vue
@@ -43,11 +43,11 @@
                   <span>{{ item }}</span>
                 </template>
               </v-select>
-              <v-text-field v-model="form.rate"
+              <v-text-field v-model="form.designation"
                             class="pa-1"
                             label="Designation (optional)"
                             placeholder="BM3" />
-              <v-text-field v-model="form.designation"
+              <v-text-field v-model="form.warfare"
                             class="pa-1"
                             label="Warfare (optional)"
                             placeholder="ESWS/SS" />
@@ -97,7 +97,6 @@ export default Vue.extend({
       middleInitial: null,
       rank: null,
       officer: null,
-      rate: null,
       designation: null,
       ssn: null,
       memberStatus: null,
diff --git a/src/components/TheSailorDetailComponent.vue b/src/components/TheSailorDetailComponent.vue
index ca767ec..fcfb373 100644
--- a/src/components/TheSailorDetailComponent.vue
+++ b/src/components/TheSailorDetailComponent.vue
@@ -3,7 +3,7 @@
     <v-toolbar dense
                flat>
       <v-toolbar-title>
-        {{ sailor.rate }} {{ sailor.lastName }}, {{ sailor.firstName }}
+        {{ sailor.rank }} {{ sailor.lastName }}, {{ sailor.firstName }}
         <v-btn icon
                small
                color="primary"
