diff --git a/src/store/modules/sailors.js b/src/store/modules/sailors.js
index 89acc2a..2b61c93 100644
--- a/src/store/modules/sailors.js
+++ b/src/store/modules/sailors.js
@@ -9,8 +9,8 @@ export default {
   },
   getters: {
     getSailorsSummaryList: state => {
-      if (state.data.length) {
-        return state.data?.map(sailor => ({
+      if (state.data && state.data.length) {
+        return state.data.map(sailor => ({
           uuid: sailor.uuid,
           name: `${sailor.lastName}, ${sailor.firstName} : ${sailor.rank}`
         }));
@@ -18,7 +18,7 @@ export default {
       return [];
     },
     getSailorById: state => givenUuid => {
-      if (state.data.length) {
+      if (state.data && state.data.length) {
         return state.data?.find(sailor => sailor.uuid === givenUuid);
       }
       return {};
