diff --git a/src/store/actions.js b/src/store/actions.js
index 8d73a6f..d50597d 100644
--- a/src/store/actions.js
+++ b/src/store/actions.js
@@ -16,8 +16,9 @@ export default {
         commit("setErrorMsg", response.error?.toString());
         commit("setErrorObj", response);
       }
-      dispatch("loadDb");
-      dispatch("setSelectedSailor", response.uuid);
+      dispatch("loadDb").then(() => {
+        dispatch("setSelectedSailor", response.uuid);
+      });
     });
   },
   updateSailor({ commit, dispatch }, payload) {
